controllers/util/common/common.go

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright 2022 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/IBM/ibm-block-csi-operator/controllers/internal/CRUtils"
+	"github.com/IBM/ibm-block-csi-operator/controllers/util"
+	oconfig "github.com/IBM/ibm-block-csi-operator/pkg/config"
+	"github.com/go-logr/logr"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ControllerHelper struct {
+	client.Client
+	Log logr.Logger
+}
+
+func NewControllerHelper(client client.Client) *ControllerHelper {
+	return &ControllerHelper{
+		Client: client,
+	}
+}
+
+func (ch *ControllerHelper) DeleteClusterRoleBindings(clusterRoleBindings []*rbacv1.ClusterRoleBinding) error {
+	logger := ch.Log.WithName("deleteClusterRoleBindings")
```

Comment:
```suggestion
	logger := ch.Log.WithName("DeleteClusterRoleBindings")
```

Reply:
done

Reply:
same for `GetAccessorAndFinalizerName`, etc...

Reply:
I still see this
![image](https://user-images.githubusercontent.com/57443811/172151754-87fdeb16-a2fa-46d1-9e65-a911c0bbd82a.png)

Reply:
I thought you meant that all the logs will start in capital letter

Reply:
done